pages/app-layout/with-sticky-header-table-in-content-layout.page.tsx

```diff
@@ -0,0 +1,37 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React from 'react';
+import AppLayout from '~components/app-layout';
+import Header from '~components/header';
+import Table from '~components/table';
+import { generateItems, Instance } from '../table/generate-data';
+import { columnsConfig } from '../table/shared-configs';
+import { Breadcrumbs } from './utils/content-blocks';
+import labels from './utils/labels';
+import ContentLayout from '~components/content-layout';
+import ScreenshotArea from '../utils/screenshot-area';
+
+const items = generateItems(20);
+
+export default function () {
+  return (
+    <ScreenshotArea gutters={false}>
```

Comment:
Visual regression testing is fine, but how about adding a normal test with assertions?

Take this one as example: https://github.com/cloudscape-design/components/blob/8b62025c8010376f5c7db65a81581b0bc2a7409a/src/container/__integ__/awsui-applayout-sticky.test.ts#L70-L83

Reply:
Added in [9992501](https://github.com/cloudscape-design/components/pull/2284/commits/9992501596cf78d7ab52b41830af3383ea1f80e9).

Checked that it fails before the changes.